ChatQnA/tests/test_compose_on_xeon.sh

```diff
@@ -178,13 +178,26 @@ function stop_docker() {
 function main() {
 
     stop_docker
+    # install perfspect
+    wget -qO- https://github.com/intel/PerfSpect/releases/latest/download/perfspect.tgz | tar xvz
+
+    # get used disk
+    list="$(./perfspect/perfspect report --filesystem 2>&1)"
+    list=$( for i in ` echo  $list `; do [[ $i =~ json ]] && echo $i ; done )
+    used_disk_before="$(python parse_perfspect.py $list 2>&1)"
+
     if [[ "$IMAGE_REPO" == "opea" ]]; then build_docker_images; fi
     start_time=$(date +%s)
     start_services
     end_time=$(date +%s)
     duration=$((end_time-start_time))
     echo "Mega service start duration is $duration s" && sleep 1s
 
+    # trace  max used memory
+    list="$(./perfspect/perfspect telemetry --duration 10 2>&1)"
```

Comment:
Line 196 to line 198 could be part of `parse_perfspect.py`, since it's common used for all examples.